### PR TITLE
Feat/33 read daily agenda

### LINKED ISSUE #33: [FEAT] 아젠다 조회시 pagingId 기준 정렬하여 반환
## ✨작성할 기능에 대한 설명을 적어주세요
```
아젠다를 조회시 반환하는 리스트를 pagindId가 큰 순으로 정렬하여 반환.
```

<br><br>

## ✅ 해야 할 작업들을 작성해주세요
- [x] dto반환시 정렬 기능 추가

<br><br>

## 🦋 연관된 이슈
-

### PR DESCRIPTION
**Pull request by OneK-2**
## ✨ 요약
```
업데이트 에러처리 수정
페이징 정렬 조회 추가
```

<br><br>

## 😎 해결한 이슈
- close #33 

<br><br>

### CODE REVIEW

**Comment by minwoo1999** [comment]
### 테스트 커버리지 측정
|File|Coverage [87.62%]|:green_apple:|
|:-|:-:|:-:|
|[DailyAgendaServiceImpl.java](https://github.com/rework-kr/REWORK-SERVER/blob/e445d308a8a8c208c328870b528a821f2bcbcbea/src%2Fmain%2Fjava%2Fcom%2Fexample%2Frework%2Fdailyagenda%2Fapplication%2Fimpl%2FDailyAgendaServiceImpl.java)|87.62%|:green_apple:|

|Total Project Coverage|72.26%|:green_apple:|
|:-|:-:|:-:|

**Comment by minwoo1999** [comment]
### 테스트 커버리지 측정
|File|Coverage [86.55%]|:green_apple:|
|:-|:-:|:-:|
|[DailyAgendaServiceImpl.java](https://github.com/rework-kr/REWORK-SERVER/blob/cf56ce894a323105fe4c26744f9433ffb2d67ee5/src%2Fmain%2Fjava%2Fcom%2Fexample%2Frework%2Fdailyagenda%2Fapplication%2Fimpl%2FDailyAgendaServiceImpl.java)|86.55%|:green_apple:|

|Total Project Coverage|72.15%|:green_apple:|
|:-|:-:|:-:|

File: src/main/java/com/example/rework/dailyagenda/application/impl/DailyAgendaServiceImpl.java [hunk 1]
```diff
@@ -53,6 +53,7 @@ public ReadDailyAgendaResponseDto readDailyAgenda(int year, int month, int day,
         }
 
         List<ReadDetailDailyAgendaResponseDto> result = agendaList.stream()
+                .sorted((a1, a2) -> Long.compare(a2.getPagingId(), a1.getPagingId()))
```
Comment (minwoo1999) [inline src/main/java/com/example/rework/dailyagenda/application/impl/DailyAgendaServiceImpl.java:56]: 람다 스트림 sorted로 정렬하셨군요 !! 좋습니다 !! 고생하셨습니다~~

**Review by minwoo1999 — APPROVED** [review]
LGTM

**Comment by klmhyeonwoo** [comment]
"빨리 배포해줘"

**Pull request merged**